cldfbench_nominalperson_cldf.py

```diff
@@ -20,63 +20,69 @@ def cmd_download(self, args):
 
         >>> self.raw_dir.download(url, fname)
         """
-        pass
+        wals_dir = self.raw_dir / 'wals'
+        if not wals_dir.exists():
+            wals_dir.mkdir()
+        rec = Record.from_doi('10.5281/zenodo.13950591')
+        rec.download_dataset(wals_dir)
 
     def cmd_makecldf(self, args):
         # Extend the schema:
-        args.writer.cldf.add_component('LanguageTable')
-        args.writer.cldf.add_columns('LanguageTable','Walscode')    # add non-standard column
-        
-        args.writer.cldf.add_component('CodeTable')
+        args.writer.cldf.add_component(
+            'LanguageTable',
+            'Walscode',
+            {'name': 'ISOcodes', 'separator': ';', 'datatype': {'base': 'string', 'format': '[a-z]{3}'}})
+
+        #args.writer.cldf.add_component('CodeTable')
         args.writer.cldf.add_component('ParameterTable')
         args.writer.cldf.add_component('ValueTable')
         args.writer.cldf.add_component('ExampleTable')
 
+        glangs = {lg.id: lg for lg in args.glottolog.api.languoids()}
```

Comment:
In `cmdf_makecldf`, a [Glottolog](https://pyglottolog.readthedocs.io/en/latest/api.html#pyglottolog.Glottolog) instance is available as `args.glottolog.api`.